task_list.sass

```diff
@@ -15,8 +15,8 @@
   cursor: default
   pointer-events: all
 
-.ql-editor ul.task-list > li.checked
+.ql-editor ul.task-list > li.checked-true
```

Comment:
Is this what parchment produces? Is it possible to retain the class just as `checked`?
Otherwise this is a breaking change.